### ISSUE TITLE
Remove the requirement of disabling VT-d in all guides

### ISSUE THREAD

**Issue by Richy-Z** [dortania/bugtracker#347]
Guide(s) in question: Dortania OpenCore Install Guide, specifically every config.plist guide

Since macOS Big Sur, Apple has added support for VT for Directed I/O (VT-d) and Kernel DMA Protection, allowing for extended Thunderbolt functionality and security. In fact, enabling VT-d is a requirement on macOS Ventura and up in order to fully enable network adapters which include built-in adapters and PCIe ones.

What is problematic however is the fact that all of the config.plist guides explicitly state that VT-d should be turned off, with no mention that this is only required for macOS < 11.

Apple states that VT-d is supported:
> Intel-based Mac computers with Intel Virtualization Technology for Directed I/O (VT-d)

> Starting in macOS 11, all Mac computers with an Apple T2 Security Chip run UEFI drivers that facilitate DMA

https://support.apple.com/en-gb/guide/security/seca4960c2b5/web

elitemacx86 posts:
> Since macOS Ventura, enabling VT-d is a must. VT-d is required for working Network Adapters, including built-in and PCIe based.

https://elitemacx86.com/threads/how-to-enable-apple-vtd-on-macos-clover-opencore.868/

> On macOS Ventura, there is a very likely chance that your Ethernet won't work, including built-in any PCIe based such as AQuantia. To fix this issue, you'll need to enable VT-d on your System. Note that some systems may require memory mapping (fixing DMAR Table).

https://elitemacx86.com/threads/common-problems-in-macos-ventura-13-0.924/

Therefore:
- Remove VT-d from the "Disable" section
- Add Kernel DMA Protection to the "Enable" section
- Add a note saying that `DisableIoMapper` should only be used on macOS <11 (Big Sur) where VT-d cannot be disabled

**Comment by RarogCmex** [comment]
My embedded USB controller brokes if I enable VT-D. I'll try to fix this by DMAR guide.
So in some systems it looks problematic.
UPDATED: DMAR guide doesn't help [guide](https://dortania.github.io/Getting-Started-With-ACPI/Universal/dmar-methods/manual.html#preparation). My motherboard (asrock z170 extreme4) USB controller is not working. Luckily I have addidional pci-e one.
I'm in doubt what causes that

**Comment by RarogCmex** [comment]
Yes, my hardware become faulty (asrock z170 extreme4), when appleVTD is enabled.

**Comment by amane1234** [comment]
From my experience, to use Intel I225-V (Asus Z690 rog), enabling Apple VT-D was required. 

So, I think it would be helpful to include a note or a mention in the OC guide that tells some devices require VT-d.

Plus, DisableIoMapperMapping quirk is also needed to fix glitches in Firefox browser and third party video apps (VLC, Smplayer,iina,Mplayer..) after wake-up from (S3 or S4) sleep.
